morph/react-dom/get-value-for-property.js

```diff
@@ -11,9 +11,16 @@ export default (node, parent) => {
         return {
           src: safe(value, node),
         }
-      } else if (key === 'isDisabled') {
+      } else if (
+        key === 'isDisabled' &&
+        node.value.value.toString().indexOf('when') > -1
```

Comment:
Not entirely happy about this 😬 . Open to suggestions!

Reply:
you have captured `node.value.value` above as `const value`, just use that and I reckon its fine